iGCS/Utils/SettingsData.m

```diff
@@ -0,0 +1,89 @@
+//
+//  SettingsData.m
+//  iGCS
+//
+//  Created by David Leskowicz on 2/14/15.
+//
+//
+
+#import "SettingsData.h"
+#import "FileUtils.h"
+
+@implementation SettingsData
+
+static NSString* const altitudeKey = @"altitudeKey";
+static NSString* const radiusKey = @"radiusKey";
+static NSString* const ceilingKey = @"ceilingKey";
+static NSString* const unitKey = @"unitKey";
+
+
+- (void) encodeWithCoder:(NSCoder *)encoder {
+    [encoder encodeDouble:self.altitude forKey:altitudeKey];
```

Comment:
Please use the encoder that matches the type of the property. double for double, int for int and so on.